client/jetpack-connect/authorize-form.jsx

```diff
@@ -90,8 +90,11 @@ class JetpackConnectAuthorizeForm extends Component {
 		);
 	}
 
-	isWCS() {
-		return 'woocommerce-services' === this.props.jetpackConnectAuthorize.queryObject.from;
+	isWoo() {
+		const wooSlugs = [ 'woocommerce-setup-wizard', 'woocommerce-services' ];
+		const jetpackConnectSource = this.props.jetpackConnectAuthorize.queryObject.from;
+
+		return -1 !== wooSlugs.indexOf( jetpackConnectSource );
```

Comment:
What do you think to using [`_.includes`](https://lodash.com/docs/4.17.4#get) here?
It would add a little syntactic sugar, like so:
```js
return includes( wooSlugs,  jetpackConnectSource );
```